boot/freeldr/freeldr/include/fs/ntfs.h

```diff
@@ -65,6 +65,8 @@
 #define NTFS_FILE_NAME_DOS            2
 #define NTFS_FILE_NAME_WIN32_AND_DOS        3
 
+#define NTFS_MFT_MASK 0x0000FFFFFFFFFFFFULL
+
```

Comment:
Checked the code in both PRs, only this line crosses more or less, others are ok :)

Reply:
The others (where the `NTFS_MFT_MASK` mask is individually applied to `IndexedFile` only when needed) is closer to what is done in our ntfs driver, than what the other PR I mentioned does.

Reply:
Btw
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=NTFS_MFT_MASK
This is duplicated in "drivers/filesystems/ntfs/ntfs.h"
Your suggestions?

Reply:
It's fine to have a copy of this value in freeldr, since we don't share FS driver code.